src/plugins/kibana_react/public/page_template/solution_nav/solution_nav_avatar.tsx

```diff
@@ -27,13 +32,30 @@ export type KibanaPageTemplateSolutionNavAvatarProps = DistributiveOmit<EuiAvata
 export const KibanaPageTemplateSolutionNavAvatar: FunctionComponent<
   KibanaPageTemplateSolutionNavAvatarProps
 > = ({ className, size, ...rest }) => {
+  const { euiTheme } = useEuiTheme();
+
+  const pageTemplateSolutionNavAvatarStyles = {
+    base: css`
+      ${css(useEuiShadow('s'))};
+    `,
+    xxl: css`
```

Comment:
@elastic/cloud-security-posture can you review and test this? The `xxl` style is only used in `src/plugins/kibana_react/public/page_template/no_data_page/no_data_page_body/no_data_page_body.tsx` and that is only used in:
* `x-pack/solutions/security/plugins/cloud_security_posture`
* `x-pack/solutions/security/plugins/cloud_defend`